src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
```

Comment:
Please include a concrete class definition with functions that throw RuntimeError("unimplemented"); 
And then please add tests that test that that exception is thrown.

Reply:
Maybe you misunderstood what I asked? Let me know if the instructions aren't clear!

Reply:
I still don't see a concrete class for this.